docs/components.rst

```diff
@@ -327,8 +327,8 @@ intent_classifier_tensorflow_embedding
 
     The algorithm also has hyperparameters to control:
         - neural network's architecture:
-            - ``num_hidden_layers_a`` and ``hidden_layer_size_a`` set the number of hidden layers and their sizes before embedding layer for user inputs;
-            - ``num_hidden_layers_b`` and ``hidden_layer_size_b`` set the number of hidden layers and their sizes before embedding layer for intent labels;
+            - ``hidden_layers_sizes_a`` sets the number of hidden layers and their sizes before the embedding layer for user inputs;
```

Comment:
let's rephrase it to smith like: `sets a list of hidden layers sizes before embedding layer for user inputs, the number of hidden layers is equal to the length of the list`